src/app/content/components/Page/PageComponent.tsx

```diff
@@ -65,11 +68,31 @@ export default class PageComponent extends Component<PagePropTypes> {
     }
 
     const highlgihtsAddedOrRemoved = this.highlightManager.update();
-    this.searchHighlightManager.update(prevProps.searchHighlights, this.props.searchHighlights, {
-      forceRedraw: highlgihtsAddedOrRemoved,
-    });
+
+    this.searchHighlightManager.update(
+      prevProps.searchHighlights,
+      this.props.searchHighlights,
+      {
+        cb: this.onHighlightSelect,
```

Comment:
can we call this option `onSelect` or something like that so it is more descriptive than `cb`